test/entity.spec.js

```diff
@@ -816,5 +818,37 @@ describe('JHipster generator for entity', () => {
                 });
             });
         });
+
+        context('reproducible build', () => {
+            describe('no dto, no service, no pagination', () => {
+                before(done => {
+                    helpers
+                        .run(require.resolve('../generators/entity'))
+                        .inTmpDir(dir => {
+                            fse.copySync(path.join(__dirname, '../test/templates/reproducible'), dir);
+                        })
+                        .withArguments(['foo'])
+                        .on('end', done);
+                });
+
+                it('creates expected default files', () => {
+                    assert.file(expectedFiles.server);
+                    assert.file(expectedFiles.clientNg2);
+                    assert.file(expectedFiles.gatling);
+                    assert.file(expectedFiles.fakeData);
+
+                    assert.fileContent(`${SERVER_MAIN_RES_DIR}config/liquibase/fake-data/foo.csv`, /1;New Leu Diverse Illinois;"0646"/);
+
+                    assert.fileContent(
+                        `${SERVER_TEST_SRC_DIR}com/mycompany/myapp/web/rest/FooResourceIT.java`,
+                        /DEFAULT_NUMBER_PATTERN_REQUIRED_TOM = "387"/
+                    );
+                    assert.fileContent(
+                        `${SERVER_TEST_SRC_DIR}com/mycompany/myapp/web/rest/FooResourceIT.java`,
+                        /UPDATED_NUMBER_PATTERN_REQUIRED_TOM = "468439"/
```

Comment:
If field name will be changed to `numberPatternRequired` then `_TOM` suffix needs to be removed.